src/CONFIG.js

```diff
@@ -49,11 +49,11 @@ export default {
     EXPENSIFY: {
         // Note: This will be EXACTLY what is set for EXPENSIFY_URL whether the proxy is enabled or not.
         EXPENSIFY_URL: expensifyURL,
-        SECURE_EXPENSIFY_URL: secureURLRoot,
         NEW_EXPENSIFY_URL: newExpensifyURL,
-        URL_API_ROOT: expensifyURLRoot,
-        STAGING_EXPENSIFY_URL: stagingExpensifyURL,
-        STAGING_SECURE_EXPENSIFY_URL: stagingSecureExpensifyUrl,
+        PRODUCTION_API_ROOT: expensifyURLRoot,
+        PRODUCTION_SECURE_API_ROOT: secureURLRoot,
+        STAGING_API_ROOT: stagingExpensifyURL,
+        STAGING_SECURE_API_ROOT: stagingSecureExpensifyUrl,
```

Comment:
The reason for renaming `URL_API_ROOT`  to `PRODUCTION_API_ROOT` is because the underlying value of `URL_API_ROOT` is always the production api, we want to make that clear and avoid confusion

The API (on some envs) can be changed dynamically. `URL_API_ROOT` variable can be mistaken to always point to the correct API for the current ENV, so instead we make it clear by renaming it to `PRODUCTION_API_ROOT` and introduce `ApiUtils.getApiRoot()`